wtxmgr/db.go

```diff
@@ -538,6 +619,41 @@ func readRawTxRecordBlock(k []byte, block *Block) error {
 	return nil
 }
 
+func extractRawTxRecordPkScript(k, v []byte, index uint32, scrLoc uint32,
```

Comment:
I don't think this should have been renamed.  As commented above, extract slices an existing slice to extract a value (still memory mapped!) while fetch returns an entirely new value or allocation.